crates/torii/core/src/sql.rs

```diff
@@ -271,6 +274,35 @@ impl Sql {
         Ok(())
     }
 
+    pub async fn set_model_member(
+        &mut self,
+        model_tag: &str,
+        entity_id: Felt,
+        is_event_message: bool,
+        member: &Member,
+        event_id: &str,
+        block_timestamp: u64,
+    ) -> Result<()> {
+        let entity_id = format!("{:#x}", entity_id);
+        let path = vec![model_tag.to_string()];
+
+        let wrapped_ty =
+            Ty::Struct(Struct { name: model_tag.to_string(), children: vec![member.clone()] });
+
+        // update model member
+        self.build_set_entity_queries_recursive(
```

Comment:
It seems to work only on simple types that are directly included in the table of the model.
Types like Array seem to not be updated.